lib/index.js

```diff
@@ -52,9 +52,26 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     asyncHandler(async (req, res) => {
       const {type} = req.params;
       const {
-        account, email, hash, clientId, serviceId,
+        account, email, hash, serviceId,
         authenticationMethod, requiredAuthenticationMethods, typeOptions
       } = req.body;
+      let {clientId} = req.body;
```

Comment:
Let's remove the option to accept this from the body and only do the cookie mechanism.